runtime-modules/storage/src/data_object_storage_registry.rs

```diff
@@ -121,6 +125,34 @@ decl_storage! {
         /// Keeps a list of storage relationships per content id.
         pub RelationshipsByContentId get(relationships_by_content_id): map T::ContentId => Vec<T::DataObjectStorageRelationshipId>;
     }
+    add_extra_genesis {
+        config(storage_working_group_mint_capacity): minting::BalanceOf<T>;
+        config(opening_human_readable_text_constraint): InputValidationLengthConstraint;
+        config(worker_application_human_readable_text_constraint): InputValidationLengthConstraint;
+        config(worker_exit_rationale_text_constraint): InputValidationLengthConstraint;
+        build(|config: &GenesisConfig<T>| {
+            // Create a mint.
+            let mint_id_result =
+                <minting::Module<T>>::add_mint(config.storage_working_group_mint_capacity, None);
+
+            if let Ok(mint_id) = mint_id_result {
+                <working_group::Mint::<T, working_group::Instance2>>::put(mint_id);
+            } else {
+                print("Failed to create a mint for the storage working group");
```

Comment:
I would add a panic! here there is no point initializing a chain spec with bad state. And this isn't runtime code so its okay to fail loudly.

Reply:
Ok.